src/Microsoft.VisualStudio.LanguageServices.Razor/RazorEditorWorker.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.VisualStudio.LanguageServices.Razor
+{
+    public class RazorEditorWorker
```

Comment:
Also consider exposing some additional things here like the current `Workspace/ProjectId/DocumentId`.